xline-client/src/clients/watch.rs

```diff
@@ -26,4 +29,267 @@ impl WatchClient {
             )),
         }
     }
+
+    /// Watches for events happening or that have happened. Both input and output
+    /// are streams; the input stream is for creating and canceling watcher and the output
+    /// stream sends events. The entire event history can be watched starting from the
+    /// last compaction revision.
+    ///
+    /// # Errors
+    ///
+    /// If the RPC client fails to send request
+    ///
+    /// # Panics
+    ///
+    /// If the RPC server returns the wrong value
+    #[inline]
+    pub async fn watch(
+        &mut self,
+        request: WatchRequest,
+    ) -> Result<(Watcher, tonic::Streaming<WatchResponse>)> {
+        let (mut request_sender, request_receiver) = channel::<xlineapi::WatchRequest>(100);
```

Comment:
Don't use a magic number. And I think 128 is better.

Reply:
done